implementations/rust/ockam/ockam_api/src/authenticator/storage/authority_member.rs

```diff
@@ -0,0 +1,79 @@
+use ockam::identity::{Identifier, TimestampInSeconds};
+use ockam_core::compat::collections::BTreeMap;
+use ockam_core::compat::str::FromStr;
+use ockam_core::{Error, Result};
+
+/// Project member stored on the Authority node
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct AuthorityMember {
+    identifier: Identifier,
+    attributes: BTreeMap<Vec<u8>, Vec<u8>>,
+    added_by: Option<Identifier>,
```

Comment:
Do we have a use case for `added_by: None`? It doesn't seem so from the code so I would restrict that option for now.

Reply:
There was, but good poing. I made it non optional